Update job service docs (#15379)
Alluxio/alluxio: docs/en/operation/User-CLI.md

### distributedCp

The `distributedCp` command copies a file or directory in the Alluxio file system distributed across workers
using the job service. By default, the command runs synchronously and the user will get a `JOB_CONTROL_ID` after the command successfully submits the job to be executed.
The command will wait until the job is complete, at which point the user will see the list of files copied and statistics on which files completed or failed.
The command can also run in async mode with the `--async` flag. Similar to before, the user will get a `JOB_CONTROL_ID` after the command successfully submits the job.
The difference is that the command will not wait for the job to finish. 

> Users can use the [`getCmdStatus`](#getCmdStatus) [`getCmdStatus`]({{ '/en/operation/User-CLI.html' | relativize_url }}#getCmdStatus)

Reviewer: only provide one, the current state will actually show up with two words with different links pointing to the same thing
```suggestion
Users can use the [`getCmdStatus`](#getCmdStatus)
```
the example i gave can also be used for anchor tags in different pages, but since it is in the same page the shorter version is preferred

### distributedCp

The `distributedCp` command copies a file or directory in the Alluxio file system distributed across workers
using the job service. In sync mode which is the default mode, user will get a JOB_CONTROL_ID after the command submits successfully.
User need to wait for the command to finish. When the command finishes, user will be able to see the file copy status and statistics on completion/failure in sync mode.
In async mode user will get a JOB_CONTROL_ID after the command submission is successful. The command execution happens asynchronously on the Job Master side.

> User can use `getCmdStatus` to check detailed status information on the JOB_CONTROL_ID.

Author: OK, thanks for the notice

> User can use `getCmdStatus` to check detailed status information on the JOB_CONTROL_ID.

Reviewer: ping on this comment. also was hoping to apply this change first before copying it over to the section for distributedLoad

A lower value means slower execution but also being nicer to the other users of the job service.
* `--overwrite`: Whether to overwrite the destination. Default is true.
* `--batch-size`: Specifies how many files to be batched into one request. The default value is `20`. Notice that if some task failed in the batched job, the whole batched job would fail with some completed tasks and some failed tasks.
* `--not-wait`: Specifies whether to wait for command execution to finish. If not explicitly shown then default to wait or run synchronously.
```console
$ ./bin/alluxio fs distributedCp --active-jobs 2000 /data/1023 /data/1024
```

### distributedLoad

The `distributedLoad` command loads a file or directory from the under storage system into Alluxio storage distributed
across workers using the job service. The job is a no-op if the file is already loaded into Alluxio.

> User will get a job control ID after the command submission is successful in async mode.

Reviewer: but sync mode is default? why explain async mode first?

Author: swapped the sequence and added more details

> In sync mode, user will get a job control Id after the command submits successfully.

Reviewer: previously used `ID`, now used `Id`??

this sentence is a bit confusing; after reading the previous one and this one, it sounds like both sync and async do the same thing. clearly explain the differences

Author: changed all to JOB_CONTROL_ID

A lower value means slower execution but also being nicer to the other users of the job service.
* `--overwrite`: Whether to overwrite the destination. Default is true.
* `--batch-size`: Specifies how many files to be batched into one request. The default value is `20`. Notice that if some task failed in the batched job, the whole batched job would fail with some completed tasks and some failed tasks.
* `--async`: Specifies whether to wait for command execution to finish. If not explicitly shown then default to run synchronously.
```console

> $ ./bin/alluxio fs distributedCp --active-jobs 2000 /data/1023 /data/1024

Reviewer: follow the same format as the console section in distributedCp? at least add an additional example line with `--async` flag

Author: follow distributedLoad, yeah, have done and updated

```

### distributedLoad

The `distributedLoad` command loads a file or directory from the under storage system into Alluxio storage distributed

> across workers using the job service. The job is a no-op if the file is already loaded into Alluxio.

Reviewer: update to something similar as distributedCp above?

The `distributedLoad` command loads a file or directory from the under storage system into Alluxio storage distributed
across workers using the job service. The job is a no-op if the file is already loaded into Alluxio.
By default, the command runs synchronously and the user will get a `JOB_CONTROL_ID` after the command successfully submits the job to be executed.
The command will wait until the job is complete, at which point the user will see the list of files loaded and statistics on which files completed or failed.
The command can also run in async mode with the `--async` flag. Similar to before, the user will get a `JOB_CONTROL_ID` after the command successfully submits the job.
The difference is that the command will not wait for the job to finish.

> Users can use the [`getCmdStatus`](#getCmdStatus) [`getCmdStatus`]({{ '/en/operation/User-CLI.html' | relativize_url }}#getCmdStatus)

Reviewer: same here

$ ./bin/alluxio fs distributedLoad /data/today --excluded-locality ROCK1,ROCK2
# Include all workers except which's locality belong to the localities in the excluded locality file
$ ./bin/alluxio fs distributedLoad /data/today --excluded-locality-file /tmp/localityfile-exclude

> # Turn on async submission mode

Reviewer: nit: extra space at the start of the line that was not present in most of the above lines

$ ./bin/alluxio fs distributedLoad /data/today --excluded-locality ROCK1,ROCK2
# Include all workers except which's locality belong to the localities in the excluded locality file
$ ./bin/alluxio fs distributedLoad /data/today --excluded-locality-file /tmp/localityfile-exclude

> # Turn on async submission mode. Run this command to get JOB_CONTROL_ID, then use `getCmdStatus` to check command detailed status.
> $ ./bin/alluxio fs distributedLoad /data/today --async

Reviewer: don't use backticks inside the code block because they will actually show up as backticks instead of code font. but more importantly, seems easier to show how to use getCmdStatus instead of describing it
```suggestion
# Turn on async submission mode. The command will output a JOB_CONTROL_ID
$ ./bin/alluxio fs distributedLoad /data/today --async
# Get the status of the running job
$ ./bin/alluxio fs getCmdStatus $JOB_CONTROL_ID
```

also this section describes many of the optional flags that may not usually be used, but async mode should probably be promoted better. so i would add this section at the end of the first console section, immediately before `Or you can include some workers ...`

Author: Done, good point, thanks!

$ ./bin/alluxio fs getCapacityBytes
```

### getCmdStatus

> The `getCmdStatus` command returns the detailed distributed command status based on a given Job Control Id.

Reviewer: `job control ID` != `job control Id` != `Job Control Id`

For example, `getCmdStatus` can be used to check what files are loaded in a distributed command, and how many succeeded or failed.

```console

> $ ./bin/alluxio fs getCmdStatus $job_control_id

Reviewer: example output?

$ ./bin/alluxio fs getCapacityBytes
```

### getCmdStatus

The `getCmdStatus` command returns the detailed distributed command status based on a given JOB_CONTROL_ID.
The detailed status includes:
1. Successfully loaded or copied file paths.
2. Statistics on the number of successful and failed file paths.
3. Failed file paths, logged in a separate csv file.

For example, `getCmdStatus` can be used to check what files are loaded in a distributed command, and how many succeeded or failed.

```console
$ ./bin/alluxio fs getCmdStatus $JOB_CONTROL_ID

> Sample Output:

Reviewer: this example output is great, would like the same for the other distributed commands if possible

Author: Added more sample outputs.
